### ISSUE TITLE
host function logging may be too verbose

### ISSUE THREAD

**Issue by bowenwang1996** [near/nearcore#5461]
Given that we log almost all the host function invocations now, neard logging feels too verbose. For example
```
Nov 25 14:31:14 testnet-canary0 sh[24750]: Nov 25 14:31:14.516 DEBUG process_state_update:Runtime::apply:Runtime::process_receipt:run_wasmer2{code.len=201136 method_name=ft_transfer_call}:run_method:run_method/call:register_len: host-function: enter
Nov 25 14:31:14 testnet-canary0 sh[24750]: Nov 25 14:31:14.516 DEBUG process_state_update:Runtime::apply:Runtime::process_receipt:run_wasmer2{code.len=201136 method_name=ft_transfer_call}:run_method:run_method/call:register_len: host-function: close time.busy=8.51µs time.idle=583ns
Nov 25 14:31:14 testnet-canary0 sh[24750]: Nov 25 14:31:14.516 DEBUG process_state_update:Runtime::apply:Runtime::process_receipt:run_wasmer2{code.len=201136 method_name=ft_transfer_call}:run_method:run_method/call:read_register: host-function: enter
Nov 25 14:31:14 testnet-canary0 sh[24750]: Nov 25 14:31:14.516 DEBUG process_state_update:Runtime::apply:Runtime::process_receipt:run_wasmer2{code.len=201136 method_name=ft_transfer_call}:run_method:run_method/call:read_register: host-function: close time.busy=8.60µs time.idle=857ns
```
I am not sure whether there is a need to log host functions such as `read_register`, `register_len`, and `value_return`. I think we should be more clear on why we need to log almost all host functions first and decide what to do accordingly. cc @matklad

**Comment by nagisa** [comment]
I would argue that `DEBUG` is not an appropriate default level to have set. I would argue that `INFO` or `WARNING` should be the most suitable default level for events in most use-cases and `DEBUG` or lower should only be set when actively investigating an issue of some sort.

That said, these look like span event logging, which `tracing-subscriber` logging implementation does not output by default, to the best of my knowledge. I'm not sure why it was enabled, but much like with the level, this kind of thing is intended for people debugging subscriber implementations, rather than the instrumented applications. Much like with the log level, I'd argue that these should only be enabled when actively investigating some specific issue.

**Comment by matklad** [comment]
>I would argue that DEBUG is not an appropriate default level to have set.

Yeah, that's not the default level, that's verbose level we use for some of our test nodes to more easily investigate issues on a live network. 

> I'm not sure why it was enabled, 

That's primarily to get timing information. Sometimes stuff runs slower in production than we'd expect, and understanding what *exactly* is slow is pretty helpful.

**Comment by matklad** [comment]
cc @nikurt as the chief logging officer. 

I see several options here: 

* decide we don't actually need these logs (but then maybe hit another incident where they'd be useful).
* decide that some frequent and obvious host functions don't require logging, and add some filtering (in a sense, that's already what we do).
* up our logging game here, print only `INFO`s to the terminal, and redirect `DEBUG` to some system which allows for easier filtering.

**Comment by bowenwang1996** [comment]
Given what @matklad suggested, assigning the issue to @nikurt :)
